client/src/components/dashboard/Edit.js

```diff
@@ -26,9 +26,7 @@ class Edit extends React.Component {
   componentDidMount() {
     $('select').material_select();
     $('#selectMe-edit').on('change', this.handleChange);
-    $('.modal').modal({
-      dismissible: false
-    });
+    $('.modal').modal();
```

Comment:
**Code Quality**

I'm curious as to why you're using jquery and manipulating the DOM directly. Remember that React follows a declarative paradigm